.github/workflows/smoke-test-release.yml

```diff
@@ -49,3 +49,53 @@ jobs:
           npx wc-e2e test:e2e ./tests/e2e/specs/smoke-tests/update-woocommerce.js
           npx wc-e2e test:e2e
           npx wc-api-tests test api
+  test-wp-version:
+    name: Smoke test on L-${{ matrix.wp }} WordPress version
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix: 
+        wp: [ '1', '2' ]
+    steps:
+
+      - name: Create dirs.
```

Comment:
Could we use the same copy of WooCommerce for all three test runs vs recreating the same setup for each run? The `docker:up` will create a fresh test install.

Reply:
Ah good call, so just to make sure, were you thinking of doing something like a setup step before each subsequent job: release smoke test run, and the matrix of the -1 and -2 WordPress versions; or just having that set up before the -1 and -2 WordPress test runs?

Reply:
In theory, we should be able to run docker:down then docker:up for L-1 and L-2 as long as we have the wp version set as an environment variable.

Reply:
I played with a few options on this @rrennick and wanted to share the results. I was looking for a good way to reuse the same WooCommerce copy between runs and working around each job working in its own environment. Just running docker down and up works great when swapping out the environment variable, but reusing the same code as a bit challenge. I tried the following:

* Using artifacts

I tested with using artifacts, something like this:

```yml
      - name: Temporarily save environment
        uses: actions/upload-artifact@v2
        with:
          name: wc-artifact
          path: package/woocommerce
          retention-days: 1
```

Then we'd retrieve it later:

```yml
      - name: Retrieve saved envrionment
        uses: actions/download-artifact@v2
        with:
          name: wc-artifact
```

Unfortunately it didn't seem to buy us much as it had to download the artifact each run.

* Using caches

This seemed like overkill and not suited for our needs here (they persist for 7 days and can't be deleted) so I didn't dig much more deeper into this. Some more details can be found in [About caching workflow dependencies](https://docs.github.com/en/actions/advanced-guides/caching-dependencies-to-speed-up-workflows#about-caching-workflow-dependencies).

Unfortunately I didn't see a good way to do a setup within the job using a matrix, so I am curious if you happened across anything that might be useful here I may have missed as a potential solution.

Reply:
> Unfortunately I didn't see a good way to do a setup within the job using a matrix

It was worth investigating this but it's not critical that we find a way to implement this. I'll resolve the conversation.